lib/neurx.ex

```diff
@@ -20,7 +20,20 @@ defmodule Neurx do
   @doc """
   Trains the network on given data.
   """
-  def train() do
+  def train(network_pid, training_data, options \\ %{}) do
```

Comment:
I like the encapsulation of the endpoints into their own folder and some parsing can be done here :+1: